tests/integration_tests/allennlp_tests/test_allennlp.py

```diff
@@ -230,7 +230,7 @@ def test_allennlp_pruning_callback() -> None:
 
         def objective(trial: optuna.Trial) -> float:
             reader = allennlp.data.dataset_readers.TextClassificationJsonReader(
-                tokenizer=allennlp.data.tokenizers.SpacyTokenizer()
```

Comment:
[Note]

This change was introduced for avoiding the probabilistic error in loading a spacy model.
`test_allennlp_pruning_callback` sometimes fails with the following error.

```
    def load_model(name, **overrides):
        """Load a model from a shortcut link, package or data path.
        name (unicode): Package name, shortcut link or model path.
        **overrides: Specific overrides, like pipeline components to disable.
        RETURNS (Language): `Language` class with the loaded model.
        """
        data_path = get_data_path()
        if not data_path or not data_path.exists():
            raise IOError(Errors.E049.format(path=path2str(data_path)))
        if isinstance(name, basestring_):  # in data dir / shortcut
            if name.startswith("blank:"):  # shortcut for blank model
                return get_lang_class(name.replace("blank:", ""))()
            if name in set([d.name for d in data_path.iterdir()]):
                return load_model_from_link(name, **overrides)
            if is_package(name):  # installed as package
                return load_model_from_package(name, **overrides)
            if Path(name).exists():  # path to model data directory
                return load_model_from_path(Path(name), **overrides)
        elif hasattr(name, "exists"):  # Path or Path-like to model data
            return load_model_from_path(name, **overrides)
>       raise IOError(Errors.E050.format(name=name))
E       OSError: [E050] Can't find model 'en_core_web_sm'. It doesn't seem to be a shortcut link, a Python package or a valid path to a data directory.
/opt/hostedtoolcache/Python/3.7.9/x64/lib/python3.7/site-packages/spacy/util.py:175: OSError
```

I didn't investigate this error in detail but it could use a simple whitespace tokenizer instead of SpacyTokenizer, which doesn't need any model resource. (Should we make another PR?)